scripts/run-component-tests.js

```diff
@@ -14,6 +14,7 @@
  *   --local    | If specified, no browser will be launched.
  *   --firefox  | Instead of Chrome being used for tests, Firefox will be used.
  *   --no-watch | Watch mode is enabled by default. This flag opts-out to standard Bazel.
+ *   --all      | Runs tests for all components in the project.
```

Comment:
Should this be a target name, rather than a flag? If it's a flag you can do weird things like `yarn test button --all`. You have validation for it below, but we could avoid it by making it a target.

Reply:
I think even if it's a target name, people could pass in more components. Also it makes the non-flag arguments ambiguous since those arguments should always be component names/paths.

I don't feel strongly about it though. This just felt easier to implement/document.

Reply:
I'm ok with the flag version